src/components/FullWidthPromo/_full-width-promo.scss

```diff
@@ -84,14 +84,13 @@
   font-size: var(--body-xs);
   margin: 0;
   text-transform: uppercase;
-}
 
-.cc-full-width-promo__type + .cc-full-width-promo__authors:before {
-  color: var(--colour-amber-30);
-  content: '/';
-  font-size: var(--body-xs);
-  padding-left: var(--space-unit);
-  padding-right: var(--space-unit);
+  &:after {
+    color: var(--colour-amber-30);
+    content: '/';
+    padding-left: var(--space-unit);
+    padding-right: var(--space-unit);
+  }
 }
```

Comment:
This obviously causes the "slash" on permanently, even when no authors (image below explains better than words) - but I think you intended this, and I also think it looks better.

![Screenshot 2021-02-03 at 15 47 20](https://user-images.githubusercontent.com/8282383/106771698-28486680-6637-11eb-9831-1b5475c09b6a.png)

Reply:
Yes that was the intention - it makes it consistent with the Image Card and Featured Promo.